### ISSUE TITLE
Take directory hit in rule into account without star notation

### ISSUE THREAD

**Issue by xstefank** [xstefank/wildfly-github-bot#73]
We shouldn't  require star notation, if the configuration says directory, take everything in it recusively

**Comment by The-Huginn** [comment]
On it

**Comment by The-Huginn** [comment]
It works that way currently. Good to close this one

**Comment by xstefank** [comment]
nope, remove the checks for '*' which are not necessary. I'll do it.

**Comment by The-Huginn** [comment]
Currently with glob I would assume the following is possible `src/*.java` isn't it? I have not dug deeper into it tough

**Comment by xstefank** [comment]
we will readd it when the feature will be requested, I don't think this will be used
